lib/ansible/modules/network/nxos/nxos_interface.py

```diff
@@ -487,12 +529,15 @@ def map_params_to_obj(module):
 
 def map_config_to_obj(want, module):
     objs = list()
+    cmd_keys = '''
```

Comment:
We  discussed this and @chrisvanheuveln won.

Reply:
Yes. Yes he did.